src/button/styles/button.css

```diff
@@ -0,0 +1,55 @@
+@import '../../common/styles/variables.css';
+
+.button {
+	background-color: var(--button-bg);
+	border: 1px solid var(--button-bg);
+	box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.25);
+	border-radius: var(--rounded-corners);
+	color: var(--button-color);
+	cursor: pointer;
+	font: var(--base-font);
+	font-weight: bold;
+	display: inline-block;
+	padding: var(--button-padding);
+	transition: all var(--short-animation-duration) ease;
+}
+
+.button:hover,
+.button:focus {
+	background-color: var(--button-hover-bg);
+	border-color: var(--button-hover-bg);
+	outline: none;
+}
+
+/* TODO: this could be a check icon */
+.button.pressed::before {
+	border-bottom: 2px solid var(--button-color);
+	border-right: 2px solid var(--button-color);
+	box-sizing: border-box;
+	content: '';
+	display: inline-block;
+	height: 1em;
+	margin: -1em 0.5em 0 0;
+	transform: rotate(45deg);
+	width: 0.5em;
+}
+
+/* TODO: this could be a down arrow icon */
+.button.popup::after {
```

Comment:
Does this work if you drop `.button` from this and other similar chained selectors?